### PR TITLE
Add abuseACL

### PR DESCRIPTION
**Pull request by AetherBlack**
# Description

Adding abuseACL from https://github.com/AetherBlack/abuseACL/

You can find examples here : https://github.com/AetherBlack/abuseACL/#examples

The tool is to be used with `dacledit` for example. It has proved its worth on boxes such as rebound on HTB and day 11 on TryHackMe's Advent of Cyber 2023.

### CODE REVIEW

File: sources/install/package_ad.sh [hunk 1]
```diff
@@ -1287,6 +1287,14 @@ function install_ntlm_theft() {
     add-to-list "ntlm_theft,https://github.com/Greenwolf/ntlm_theft,A tool for generating multiple types of NTLMv2 hash theft files"
 }
 
+function install_abuseACL() {
+    colorecho "Installing abuseACL"
+    python3 -m pipx install git+https://github.com/AetherBlack/abuseACL/
```
Comment (Dramelac) [inline sources/install/package_ad.sh:1292]: You can now use directly `pipx install` to use the same command everywhere

File: sources/install/package_ad.sh [hunk 2]
```diff
@@ -1287,6 +1287,14 @@ function install_ntlm_theft() {
     add-to-list "ntlm_theft,https://github.com/Greenwolf/ntlm_theft,A tool for generating multiple types of NTLMv2 hash theft files"
 }
 
+function install_abuseACL() {
```
Comment (Dramelac) [inline sources/install/package_ad.sh:1290]: It seems like you don't need any alias, please add the following line to whitelist this behavior:
`# CODE-CHECK-WHITELIST=add-aliases`
More info on the contributors doc: https://exegol.readthedocs.io/en/latest/community/contributors.html#adding-a-new-tool

**Review by Dramelac — APPROVED** [review]
Thank you for the PR! Some minor update to match our guideline and unify how we use pipx and it should be good to go

**Review by Dramelac — CHANGES_REQUESTED** [review]
Wrong action

**Comment by Dramelac** [comment]
I've editing the change myself to run the test now

**Review by Dramelac — APPROVED** [review]

**Comment by Dramelac** [comment]
Your test command failed :/, be sure your command has a return code 0.
We cannot merge until then.

**Comment by AetherBlack** [comment]
Yeah, I made a typo, my bad...

**Review by ShutdownRepo — APPROVED** [review]

**Comment by ShutdownRepo** [comment]
Bypassing CI/CD checks, we should be good to go
Merging now, if anything fails on abuseACL we'll see it on the CI/CD checks running for the upcoming release, and I'll come back here if we have an issue of some kind
Thank you for the addition @AetherBlack

**Pull request merged**